KerbalStuff/ckan.py

```diff
@@ -1,7 +1,8 @@
 import json
 import os
 import re
-import subprocess
+import threading
+import requests
```

Comment:
The import statements could need some cleanup
Unfortunately, GitHub doesn't let me do a multi-line "suggestion", so here's a diff.
```diff
diff --git a/KerbalStuff/ckan.py b/KerbalStuff/ckan.py
index d50d545..e39553d 100644
--- a/KerbalStuff/ckan.py
+++ b/KerbalStuff/ckan.py
@@ -1,12 +1,7 @@
-import json
-import os
-import re
 import threading
 import requests
 
 from flask import url_for
-# noinspection PyPackageRequirements
-from github import Github  # the package is called PyGithub
 
 from .config import _cfg
```